createdisk-library.sh

```diff
@@ -75,6 +75,8 @@ function create_qemu_image {
 
     sparsify ${destDir} ${base} ${overlay}
 
+    chmod 0644 ${destDir}/${CRC_VM_NAME}.qcow2
```

Comment:
@cfergeau then it should be `chmod 0644 ${destDir}/${overlay}.qcow2` because till now there is no `${CRC_VM_NAME}.qcow2`